src/python/metrics/logs.py

```diff
@@ -317,12 +331,37 @@ def get_source_location():
   return 'Unknown', '-1', 'Unknown'
 
 
+def _add_appengine_trace(extras):
+  """Add App Engine tracing information."""
+  if sys.version_info.major != 3 or not _is_running_on_app_engine():
+    # TODO(ochang): Remove Python 3 check once all migrated to Python 3.
```

Comment:
Does it break anything on python 2, if not can simplify this check and one above to remove sys.version_info.major != 3

Reply:
Want to avoid any unexpected breakages there. Will be removed very soon anyway.